cypress/integration/tests/developer/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -46,8 +46,8 @@ describe("Source Analysis", { tags: "@tier1" }, () => {
         deleteApplicationTableRows();
 
         //Disable all proxy settings
-        let proxy = new Proxy(data.getRandomProxyData());
-        proxy.disableProxy();
+        // let proxy = new Proxy(data.getRandomProxyData());
+        // proxy.disableProxy();
```

Comment:
Why is this commented ? 
If not needed , please remove .

Reply:
Fixed